shared/reducers/chat2.js

```diff
@@ -722,8 +725,15 @@ const rootReducer = (
         })
       )
     }
-    case Chat2Gen.updateTypers: {
-      return state.set('typingMap', action.payload.conversationToTypers)
+    case EngineGen.chat1NotifyChatChatTypingUpdate: {
```

Comment:
woot, this is possible now. before the saga would have to register an incoming handler, then we'd dispatch another action